2024-11-30 11:04:23,433 : INFO : root : Loading config.
2024-11-30 11:04:23,436 : INFO : YamlMultifileConfigLoader : Machine config file #1: config.yaml
2024-11-30 11:04:23,437 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/a187413b8cf7049e0b91852f54cd1afa.mpf_cache
2024-11-30 11:04:23,498 : WARNING : ConfigProcessor : Config file in cache changed: /Users/C_Alan/pinball/hauntedhouse/modes/attract/config/attract.yaml
2024-11-30 11:04:23,498 : INFO : ConfigProcessor : Loading config from file /Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/modes/attract/config/attract.yaml.
2024-11-30 11:04:23,500 : INFO : ConfigProcessor : Loading config: /Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/modes/attract/config/attract.yaml
2024-11-30 11:04:23,500 : INFO : ConfigProcessor : Loading config from file /Users/C_Alan/pinball/hauntedhouse/modes/attract/config/attract.yaml.
2024-11-30 11:04:23,500 : INFO : ConfigProcessor : Loading config: /Users/C_Alan/pinball/hauntedhouse/modes/attract/config/attract.yaml
2024-11-30 11:04:23,501 : INFO : ConfigProcessor : Config file cache created: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/2e30198d1c841d95b9b3e69c8a0bb49b.mpf_cache
2024-11-30 11:04:23,502 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/40928215bbb65374348eff3d4cd7b8bc.mpf_cache
2024-11-30 11:04:23,502 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/167ddff969183ffa73a4682c84ddf638.mpf_cache
2024-11-30 11:04:23,503 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/2e30198d1c841d95b9b3e69c8a0bb49b.mpf_cache
2024-11-30 11:04:23,503 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/f3e7df0542012602a298ddb9f31df321.mpf_cache
2024-11-30 11:04:23,504 : INFO : ConfigProcessor : Loading config from file /Users/C_Alan/pinball/hauntedhouse/modes/attract/shows/./upper_rotate_show.yaml.
2024-11-30 11:04:23,508 : INFO : ConfigProcessor : Loading config: /Users/C_Alan/pinball/hauntedhouse/modes/attract/shows/./upper_rotate_show.yaml
2024-11-30 11:04:23,509 : INFO : ConfigProcessor : Config file cache created: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/df2857c909c4a913cc732702f1a197bb.mpf_cache
2024-11-30 11:04:23,509 : INFO : ConfigProcessor : Loading config from cache: /var/folders/lw/36nc9wln17qdc1y8dnh6d85h0000gr/T/1ab5c5e749213b44f20a4087626085fd.mpf_cache
2024-11-30 11:04:23,510 : INFO : Machine : Mission Pinball Framework Core Engine v0.80.0.dev5
2024-11-30 11:04:23,510 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'force_assets_load': False, 'platform_integration_test': False, 'jsonlogging': False, 'logfile': 'logs/2024-11-30-11-04-23-mpf-Macpro.local.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 15, 'consoleloglevel': 20, 'force_platform': 'virtual', 'syslog_address': None, 'full_logfile_path': '/Users/C_Alan/pinball/hauntedhouse/logs/2024-11-30-11-04-23-mpf-Macpro.local.log'}
2024-11-30 11:04:23,510 : INFO : Machine : MPF path: /Users/C_Alan/envp311/lib/python3.11/site-packages/mpf
2024-11-30 11:04:23,510 : INFO : Machine : Machine path: /Users/C_Alan/pinball/hauntedhouse
2024-11-30 11:04:23,510 : INFO : Machine : Platform: darwin
2024-11-30 11:04:23,510 : INFO : Machine : Python executable location: /Users/C_Alan/envp311/bin/python3.11
2024-11-30 11:04:23,510 : INFO : Machine : Python version: 3.11.0 (64-bit)
2024-11-30 11:04:23,511 : INFO : Machine : Initializing MPF...
2024-11-30 11:04:23,588 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,602 : INFO : EventManager : Event: ======'machine_var_master_volume'====== Args={'value': 0.5, 'prev_value': None, 'change': True}
2024-11-30 11:04:23,602 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 14900, 'prev_value': None, 'change': True}
2024-11-30 11:04:23,602 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev5', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,602 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'Mission Pinball Framework v0.80.0.dev5, Config version:6, Show version: 6, BCP version:1.1', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,602 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.11.0', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'macOS-14.6.1-x86_64-i386-64bit', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Darwin', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '23.6.0', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': 'Darwin Kernel Version 23.6.0: Mon Jul 29 21:13:00 PDT 2024; root:xnu-10063.141.2~1/RELEASE_X86_64', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'x86_64', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,620 : INFO : EventManager : Event: ======'machine_var_log_file_path'====== Args={'value': '/Users/C_Alan/pinball/hauntedhouse/logs/2024-11-30-11-04-23-mpf-Macpro.local.log', 'prev_value': None, 'change': True}
2024-11-30 11:04:23,621 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2024-11-30 11:04:23,664 : INFO : EventManager : Event: ======'drop_target_bank_upper_up'====== Args={}
2024-11-30 11:04:23,664 : INFO : EventManager : Event: ======'drop_target_bank_lower_up'====== Args={}
2024-11-30 11:04:23,664 : INFO : EventManager : Event: ======'init_phase_2'====== Args={}
2024-11-30 11:04:23,669 : ERROR : asyncio : Exception in callback EventManager._queue_task_done(<Task finishe...:shows None')>)
handle: <Handle EventManager._queue_task_done(<Task finishe...:shows None')>)>
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/events.py", line 795, in _queue_task_done
    future.result()
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/config_players/device_config_player.py", line 44, in validate_config_entry
    raise AssertionError(
AssertionError: Invalid settings for player mode_attract_started.1:shows None
2024-11-30 11:04:23,672 : INFO : Machine : Starting the main run loop with active modes: []
2024-11-30 11:04:23,672 : WARNING : Machine : Attract mode is not active, game will not be playable. Please check your attract mode configuration.
2024-11-30 11:04:23,672 : INFO : Machine : Shutting down...
2024-11-30 11:04:23,672 : INFO : EventManager : Event: ======'shutdown'====== Args={}
2024-11-30 11:04:24,673 : ERROR : Machine : Runtime Exception
Traceback (most recent call last):
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/machine.py", line 808, in _run_loop
    raise self._exception['exception']
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/events.py", line 795, in _queue_task_done
    future.result()
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/events.py", line 708, in _run_handlers_sequential
    handler.callback(queue=queue, **merged_kwargs)
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/mode_controller.py", line 97, in initialize_modes
    item.method(config=mode.config[item.config_section],
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/config_player.py", line 150, in process_mode_config
    config = self.validate_config(config)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/core/config_player.py", line 113, in validate_config
    validated_config[event] = self.validate_config_entry(settings, event)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/C_Alan/envp311/lib/python3.11/site-packages/mpf/config_players/device_config_player.py", line 44, in validate_config_entry
    raise AssertionError(
AssertionError: Invalid settings for player mode_attract_started.1:shows None
2024-11-30 11:04:24,674 : INFO : root : MPF run loop ended.
